README.md

```diff
@@ -42,12 +50,13 @@ You may obtain a copy of the License at
 - 🔎 **Detection**: detect whether a text has been generated by an LLM.
 - 🕵️‍♂️ **Attribution**: identify what LLM has generated a text.
 - 🚧 **Boundary detection**: find the boundary between human and generated text.
+- 🎨 **Mixcase**: ascertain whether specific text spans are human-written or generated by LLMs.
```

Comment:
Does using the name 'mixcase' make sense? I think it was only  used in one paper. WDYT about just using "mixed" instead?

Reply:
hmm, not sure. There is no convention right now to name this task, so I would prefer to use the name appearing in the paper you mention rather than using an alternative name. In any case, we can change the names in the future.